datajoint/autopopulate.py

```diff
@@ -118,12 +122,12 @@ def populate(self, *restrictions, suppress_errors=False, reserve_jobs=False, ord
                             jobs.complete(self.target.table_name, key)
         return error_list
 
-    def progress(self, restriction=None, display=True):
+    def progress(self, *restrictions, display=True):
```

Comment:
I am hesitant to do that since `progress` has the same signature as `populate`, which takes a lot more named arguments.